fdbserver/OldTLogServer_6_2.actor.cpp

```diff
@@ -2813,6 +2813,16 @@ ACTOR Future<Void> restorePersistentState(TLogData* self,
 		removed.push_back(errorOr(logData->removed));
 		logsByVersion.emplace_back(ver, id1);
 
+		TraceEvent ev("Role", recruited.id());
```

Comment:
We might consider using a different event name than `Role` to avoid overloading its purpose. There are some queries that make use of the `Role` field, and this could potentially complicate them.

Reply:
Yes. I think I can add the epoch to TLogPersistentStateRestore event.